docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -33,64 +46,80 @@ out = traced_model(example_input)
 
 The process of tracing takes an example input and traces its flow through the model. You can trace the model by creating an example image input, as shown in the above code using random data. To understand the reasons for tracing and how to trace a PyTorch model, see [Model Tracing](model-tracing).
 
-```{admonition} Set the Model to Evaluation Mode
+If your model uses a data-dependent control flow, such as a loop or conditional, the traced model won't generalize to other inputs. In such cases you can experiment with applying PyTorch's JIT script ([`torch.jit.script`](https://pytorch.org/docs/stable/generated/torch.jit.script.html)) to your model as described in [Model Scripting](model-scripting). You can also use a combination of tracing and scripting.
 
-To ensure that operations such as dropout are disabled, it's important to set the model to evaluation mode (_not_ training mode) before tracing. This setting also results in a more optimized version of the model for conversion.
+### ExportedProgram
+
+[ExportedProgram](https://pytorch.org/docs/stable/export.html) is a new ntermediate representation of a PyTorch model introduced in PyTorch 2. To generate an ExportedProgram representation from PyTorch code, use PyTorch's exporter ([`torch.export.export`](https://pytorch.org/docs/stable/export.html#torch.export.export)) to _export_ the model, as shown in the following example:
+
+```python
+# Export the model with random data.
+example_inputs = (torch.rand(1, 3, 224, 224),)
+exported_program = torch.export.export(torch_model, example_inputs)
 ```
 
-If your model uses a data-dependent control flow, such as a loop or conditional, the traced model won't generalize to other inputs. In such cases you can experiment with applying PyTorch's JIT script ([`torch.jit.script`](https://pytorch.org/docs/stable/generated/torch.jit.script.html)) to your model as described in [Model Scripting](model-scripting). You can also use a combination of tracing and scripting.
+Core ML Tools torch.export conversion support is newly added in 8.0, currently in beta state, in line with the export API status in PyTorch. As of Core ML Tools 8.0, representative models such as mobile bert, resnet, vit, [mobile net](convert-a-torchvision-model-from-pytorch), [deep lab](convert-a-pytorch-segmentation-model), [openelm](convert-openelm) can be converted, and total torch op translation test coverage is roughly ~60%. We would recommend to use this actively developing path
```

Comment:
`We would recommend to use this actively developing path` : this sentence doesn't make sense. 
If its in active development, why is it recommended? 

I think for now we can skip this line. Once we have more tests passing, then we can add a line here, sth to the effect of,   to try this path (as jit.trace is in maintenance mode) and if any issues are encountered (e.g. lower performance compared to jit.trace path), to report it on Github via an issue.

Reply:
Rephrased to
```
We would recommend to start trying this path, as PyTorch is moving away from torch.jit.trace
```